### PR TITLE
Doc/update doc 1.2

### PR DESCRIPTION
**Pull request by steven-zou**

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -19,7 +19,11 @@ Weiwei He , heww, VMware
...
 Canfeng Wang , wangcanfengxs , Netease
 Huijun Zhu , Ghostbaby , Yunmanman
-Xiaoyang Zhu , soulseen , QingCloud
-Chengyu Zhang , chlins , AlaudaCloud
+Xiaoyang Zhu , soulseen , Shoppee
+Chengyu Zhang , chlins , VMware
 Xavier Duthil , XavierDuthil , OVHcloud
+Yinan Li , cndoit18 , Qihu360
+Lubron Zhan , lubronzhan , VMware
+Xiongxiong Yuan, yxxhero, DiDi
```
Comment (yxxhero) [inline MAINTAINERS:27]: Qunar yxxhero

File: README.md [hunk 2]
```diff
@@ -24,30 +24,26 @@ Harbor deployment stack is controlled by a custom Harbor resource `HarborCluster
...
-  * [ ] [oss](https://www.alibabacloud.com/product/oss): A driver storing objects in Aliyun OSS.
-  * [ ] [gcs](https://cloud.google.com/storage): A driver storing objects in a Google Cloud Storage bucket.
-* Supports updating the deployed Harbor cluster
-  * Remove the optional Harbor components
-* More day2 operations (see [PoC project](https://github.com/goharbor/harbor-operator))
-  * Auto mapping Kubernetes namespaces and Harbor project
-  * Pull secrets injections
-  * Container image path rewriting
-  * Transparent proxy cache settings
+* Support services exposed with LoadBalancer
```
Comment (cndoit18) [inline README.md:46]: LoadBalancer is not supported

File: docs/feature-areas.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+## Owners of technical pillars
+
+|    Feature area        |                   Owners                 |
+|------------------------|------------------------------------------|
+| CI/CD & Development    | [ZiMing Zhang](https://github.com/bitsf) |
+| K8s CRD                | [CanFeng Wang](https://github.com/wangcanfengxs) / [ChenYu Zhang](https://github.com/chlins) / [YiNan Li](https://github.com/cndoit18)|
+| Harbor                 | [ZiMing Zhang](https://github.com/bitsf) / [Weiwei He](https://github.com/heww) |
+| Day2 operations        | [Lubron Zhan](https://github.com/lubronzhan) / [ChenYu Zhang](https://github.com/chlins) |
+| Underlying operators   | [HuiJun Zhu](https://github.com/Ghostbaby) / [ChenYu Zhang](https://github.com/chlins) / [XiaoYang Zhu](https://github.com/soulseen)|
+| Reconciler             | [YiNan Li](https://github.com/cndoit18) / [Henry Kang](https://github.com/HenryKamg) / [Weiwei He](https://github.com/heww)|
```
Comment (cndoit18) [inline docs/feature-areas.md:20]: Remove about Henry Kang

File: README.md (hunk 2 above)
Comment (steven-zou) [reply README.md:46]: @cndoit18 

That's future work.

File: docs/feature-areas.md (hunk 3 above)
Comment (steven-zou) [reply docs/feature-areas.md:20]: Will remove him in next doc PR.

**Pull request merged**